src/odbc/unittests/rowset.c

```diff
@@ -12,6 +12,20 @@ test_err(int n)
 	}
 }
 
+#ifdef _MSC_VER
+/* See https://learn.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-170 */
+#pragma warning(push)
+#pragma warning(disable:4996)
+static SQLRETURN
+_CHKSetStmtOption(SQLUSMALLINT option, SQLULEN value, const char* res)
```

Comment:
Underscore capital is reserved in C99.

That's weird having macro in `common.h` and redefinition here. Why not having 
```c
SQLRETURN SQLSetStmtOption_nowarning(SQLHSTMT hstmt, SQLSMALLINT option, SQLPOINTER param);
#define CHKSetStmtOption(a,b,res) \
        CHKR2(SQLSetStmtOption_nowarning, (odbc_stmt,a,b), SQL_HANDLE_STMT, odbc_stmt, res)
```
in `common.h` and
```c
#ifdef _MSC_VER
/* See https://learn.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-170 */
#pragma warning(push)
#pragma warning(disable:4996)
#endif
SQLRETURN
SQLSetStmtOption_nowarning(SQLHSTMT hstmt, SQLSMALLINT option, SQLPOINTER param)
{
	return SQLSetStmtOption(hstmt, option, param);
}
#ifdef _MSC_VER
#pragma warning(pop)
#endif
```
in `common.c` ?

Reply:
> Underscore capital is reserved in C99.

Ah, right.  FWIW, I'd been following the lead of all the internal `_SQL*` functions, which I suppose should be renamed to respect that reservation.

At any rate, yeah, that approach looks like it should work, though perhaps it would be better to leave `CHKSetStmtOption` as is and add a `CHKSetStmtOption_nowarning` so the test could explicitly acknowledge its use of a deprecated call.

Reply:
> > Underscore capital is reserved in C99.
> 
> Ah, right. FWIW, I'd been following the lead of all the internal `_SQL*` functions, which I suppose should be renamed to respect that reservation.

Yes, some code in FreeTDS is pretty old. C99 was surely not a thing.

> At any rate, yeah, that approach looks like it should work, though perhaps it would be better to leave `CHKSetStmtOption` as is and add a `CHKSetStmtOption_nowarning` so the test could explicitly acknowledge its use of a deprecated call.

so adding
```c
SQLRETURN SQLSetStmtOption_nowarning(SQLHSTMT hstmt, SQLSMALLINT option, SQLPOINTER param);
#define CHKSetStmtOption_nowarning(a,b,res) \
        CHKR2(SQLSetStmtOption_nowarning, (odbc_stmt,a,b), SQL_HANDLE_STMT, odbc_stmt, res)
```
in `common.h`. Sounds good to me.